src/client/util.js

```diff
@@ -16,19 +17,21 @@ export function isAndroidApp () {
   return typeof AndroidBridge !== 'undefined'
 }
 
-/* usage:
- * var connectWebViewJavascriptBridge = require('webViewJavascriptBridge')
- * connectWebViewJavascriptBridge(function(bridge) {
- *   bridge.send(...)
- * }
- */
-export function connectWebViewBridge (callback) {
-  if (window.WebViewJavascriptBridge) {
-    callback(window.WebViewJavascriptBridge)
-  } else {
-    document.addEventListener('WebViewJavascriptBridgeReady', function () {
-      callback(window.WebViewJavascriptBridge)
-    }, false)
+export function calliOSBridge (message, callback) {
```

Comment:
I vote to keep it as it is.
I think we'll always want to have two different functions, one for Android and one for iOS, partly because Android has way richer js bridge functionality which makes the code in the android app simpler, but would be hard/defeat the point to reduce it to the same interface as the iOS bridge, and partly because there are times when we may want to call one and not the other. 

these changes won't impact the android situation. The Android bridge is the same and being called in the same way for Filepicker. 
There are likely some little bugs in the android app that need ironing out with the new release.